vm/interpreter/src/default_runtime.rs

```diff
@@ -296,7 +296,7 @@ where
         value: &TokenAmount,
     ) -> Result<Serialized, ActorError> {
         let msg = UnsignedMessage::builder()
-            .to(to.clone())
+            .to(*to)
```

Comment:
doesn't matter at all really, going to avoid changing in this PR because that will make other changes necessary unrelated to this, but yeah that can be done